app/views/cases/index.html.erb

```diff
@@ -76,24 +76,19 @@
           <th>Case Notes</th>
           <th></th>
         </tr>
+        <% @urgent_cases.each do |c| %>
         <tr>
-          <td>202-345-6677</td>
-          <td>Sue F</td>
-          <td>15 weeks</td>
-          <td>(sample case notes. something or other. detail)</td>
-          <td><span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span></td>
-        </tr>
-        <tr>
-          <td>202-345-6677</td>
-          <td>Sue F</td>
-          <td>15 weeks</td>
+          <td><%= c.patient.primary_phone %></td>
+          <td><%= c.patient.name %></td>
+          <td><%= "#{(DateTime.now - c.last_menstrual_period_time).to_i/7} weeks" %></td>
           <td>(sample case notes. something or other. detail)</td>
-          <td><span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span></td>
+          <td><%= link_to "Call", "#new_call", data: {toggle: "modal"} %></td>
+          <%= render partial: 'calls/new_call', locals: {c: c} %>
```

Comment:
I think the core idea here is really strong and this works very smoothly! Nice work!

A thing I noticed when clicking thru is that this always brings up the info for Patient 2, because the id for every modal is the same. If possible, do you mind tweaking this so the proper modals show up for a given individual?

Two different approaches:

a) You have a `#new_call` id assigned to the modal partial; incorporate in case ids to the `link_to` and the id of the modal (so it links to `#new_call_4` and hits a modal with id `#new_call_4`, for example).

b) Slightly more challenging and javascript intensive: Limit the page to one hidden modal at the bottom, and when you hit, replace the `#modal-body` content. This way, if there are ever a ton of urgent cases or calls this keeps the HTML on the page more lean.